### PR TITLE
feat: add the ability to change the name and code of spot

### LINKED ISSUE #10926: [Feature]: Governance proposal to change market name for spots
backport https://github.com/vegaprotocol/vega/issues/10143

### PR DESCRIPTION
**Pull request by ze97286**
Closes #10926

### CODE REVIEW

**Review by EVODelavega — DISMISSED** [review]

**Review by peterbarrow — DISMISSED** [review]

File: protos/sources/vega/governance.proto [hunk 1]
```diff
@@ -246,8 +246,18 @@ message UpdateSpotMarketConfiguration {
   LiquidityFeeSettings liquidity_fee_settings = 5;
   // The market tick size defines the minimum change in quote price for the market
   string tick_size = 6;
+  // Specifies the name and code of teh spot instrument.
```
Comment (candida-d) [inline protos/sources/vega/governance.proto:249]:
```suggestion
  // Specifies the name and code of the spot instrument.
```

File: datanode/gateway/graphql/schema.graphql [hunk 2]
```diff
@@ -4969,7 +4969,14 @@ type UpdateSpotMarket {
   updateSpotMarketConfiguration: UpdateSpotMarketConfiguration!
 }
 
+type UpdateSpotInstrumentConfiguration {
+  code: String!
```
Comment (candida-d) [inline datanode/gateway/graphql/schema.graphql:4973]:
```suggestion
  "Instrument code, human-readable shortcode used to describe the instrument."
  code: String!
```

File: datanode/gateway/graphql/schema.graphql [hunk 3]
```diff
@@ -4969,7 +4969,14 @@ type UpdateSpotMarket {
...
+  name: String!
```
Comment (candida-d) [inline datanode/gateway/graphql/schema.graphql:4974]:
```suggestion
 "Instrument name"
  name: String!
```

**Review by candida-d — DISMISSED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -51,6 +51,7 @@
 - [10831](https://github.com/vegaprotocol/vega/issues/10831) - Add API for `notionalTimeWeightedAveragePosition`.
 - [10879](https://github.com/vegaprotocol/vega/issues/10879) - Upgrade to cometbft 0.38.6.
 - [10885](https://github.com/vegaprotocol/vega/issues/10885) - Allow margin factor to be any positive.
+- [10926](https://github.com/vegaprotocol/vega/issues/10926) - Backport governance proposal to change market name for spots
```
Comment (EVODelavega) [inline CHANGELOG.md:54]: should go under v0.76.0

**Review by EVODelavega — APPROVED** [review]

**Review by candida-d — APPROVED** [review]

**Pull request merged**